src/daemon/control.rs

```diff
@@ -726,10 +740,212 @@ pub fn watchtowers_status(revaultd: &RevaultD) -> Vec<ServerStatus> {
             });
         }
     }
-
     watchtowers
 }
 
+// FIXME: probably this function should be in revault_tx, not here...
+fn build_cpfp_tx(
```

Comment:
This definitely needs a unit test.

Reply:
In the todo list already, working on it :)